index.bs

```diff
@@ -627,7 +648,64 @@ TODO: what about the service worker API? Depends on what we're doing for service
 
 <p class="note">The other interesting method, {{IdleDetector/requestPermission()|IdleDetector.requestPermission()}}, is gated on [=transient activation=]. However, even if permission was previously granted for the origin in question, we delay starting any idle detectors while prerendering.
 
-<h4 id="activation-gated">Activation-gated APIs</h4>
+<h4 id="patch-clipboard-apis">Clipboard APIs</h4>
+
+<div algorithm="Clipboard write patch">
+  Modify the {{Clipboard/write()}} method steps by inserting the following steps after the initial creation of |p|:
+
+  1. If [=this=]'s [=relevant global object=]'s [=Window/browsing context=] is a [=prerendering browsing context=], then append the following steps to [=this=]'s [=platform object/post-prerendering activation steps list=] and return |p|.
+</div>
+
+<div algorithm="Clipboard writeText patch">
+  Modify the {{Clipboard/writeText()}} method steps by inserting the following steps after the initial creation of |p|:
+
+  1. If [=this=]'s [=relevant global object=]'s [=Window/browsing context=] is a [=prerendering browsing context=], then append the following steps to [=this=]'s [=platform object/post-prerendering activation steps list=] and return |p|.
+</div>
+
+<h4 id="patch-wake-lock">Screen Wake Lock API</h4>
+
+<div algorithm="WakeLock request patch">
+  Modify the {{WakeLock/request()}} method steps by inserting the following steps after the mid-algorithm creation of |promise|:
+
+  1. If [=this=]'s [=relevant global object=]'s [=Window/browsing context=] is a [=prerendering browsing context=], then append the following steps to [=this=]'s [=platform object/post-prerendering activation steps list=] and return |promise|.
```

Comment:
Just curious: The algorithm request() algorithm uses the browsing context of the "responsible document of the current settings object". Is this another way to get the same browsing context as "this's relevant global object's browsing context"? When we patch these specs should we make these consistent?

Reply:
Following the definitions, they are indeed the same, although it's not obvious. I can work on fixing Fetch to use this. (We'd eventually like to remove "responsible document" per https://github.com/whatwg/html/issues/4335.)